keras_cv/keypoint/utils.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utility functions for keypoint transformation."""
+import tensorflow as tf
+
+
+def discard_out_of_image(keypoints, image):
```

Comment:
I wonder if we can craft a more idiomatic name for this.

maybe:
```
keypoints = keypoint.filter_invalid(keypoints, image)
```
thoughts?

Reply:
I would says `filter_out_of_image` or `filter_out_of_boundaries`. `invalid` can mean lot of things. For instance, a keypoint can become invalid if another paired keypoint is out of the image in certain scenarios.

Reply:
ok, yeah I think you make a great point.  I like filter_out_of_image